packages/x-tree-view/src/internals/plugins/useTreeViewItems/useTreeViewItems.tsx

```diff
@@ -272,8 +272,11 @@ export const useTreeViewItems: TreeViewPlugin<UseTreeViewItemsSignature> = ({
       areItemUpdatesPrevented,
     },
     contextValue: {
-      disabledItemsFocusable: params.disabledItemsFocusable,
-      indentationAtItemLevel: experimentalFeatures.indentationAtItemLevel ?? false,
+      items: {
```

Comment:
Almost all the `contextValue` properties are now scoped by plugin.
This is not a BC, the context is never directly exposed.